server/services/photoService.ts

```diff
@@ -11,14 +11,14 @@ export default class PhotoService {
     return prisonApiClient.getImageDetail(prisonerPhotoId)
   }
 
-  async getAllFacialPhotos(prisonerNumber: string, clientToken: string) {
+  async getAllFacialPhotos(prisonerNumber: string, prisonerPhotoId: number, clientToken: string) {
```

Comment:
Maybe as a follow up it might be nice to call this `mainPrisonerPhotoId` to make it clear what the intention is behind passing the id in?

Reply:
Yeah - that makes sense. I'll do that in my next follow up